docs/Reference/Limits.md

```diff
@@ -12,6 +12,17 @@ Evergreen limits users to two never expiring spawn hosts at a time.
 Exceptions can be requested on a case-by-case which will be granted based on
 [our policy](https://mongodb.stackenterprise.co/questions/1122).
 
+## Task TTL 
+
+Tasks expire expire 365 days after creation. Expired tasks will not be available through Evergreen's API or UI, but will continue to be available in [Trino](../Project-Configuration/Evergreen-Data-for-Analytics).
```

Comment:
I would maybe specify that only finished tasks (tasks than ran) are available historically in Trino.